### DISCUSSION TITLE
Small reader tool for LDM meeting notes

### DISCUSSION THREAD

**Discussion by ashmind** [dotnet/csharplang#6698] [Show and tell]
Hi all — wanted to share a small tool I've made to simplify reading of the LDM notes.
The main purpose is to clearly track what's read and what's new/unread — a bit hard to do in GitHub UI.

Should mostly work on mobile (not amazing, but usable).

Link: https://ashmind.github.io/csharplang-reader.

**Comment by iam3yal** [comment]
@ashmind Just wanted to say thank you for everything you do for the C# community, love your tools. :)
